com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,257 @@
+{
+
+  "app-id": "com.openmusicplayer.OMP",
+
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+
+  "sdk": "org.gnome.Sdk",
+
+
+  "finish-args":
+  [
+
+    "--socket=x11",
+    "--share=network",
+    "--filesystem=host",
+    "--socket=pulseaudio"
+
+  ],
+
+  "build-options" :
+  { 
+
+    "cxxflags": "-g -std=c++17 -lstdc++fs"
+
+   },
+
+  "command": "omp",
+
+  "modules":
+  [
+
+    {
+
+      "name": "taglib",
+      "buildsystem": "cmake-ninja",
+      "config-opts": ["-DENABLE_TESTING=OFF" , "-DBUILD_SHARED_LIBS=ON", "-DCMAKE_INSTALL_PREFIX:PATH=/app"],
```

Comment:
I may be wrong, but it is probably not needed: `-DCMAKE_INSTALL_PREFIX:PATH=/app`